app/src/interfaces/input-rich-text-md/index.ts

```diff
@@ -24,6 +24,117 @@ export default defineInterface({
 				},
 			},
 		},
+		{
+			field: 'editorFont',
+			name: '$t:interfaces.input-rich-text-md.editorFont',
+			type: 'string',
+			meta: {
+				width: 'half',
+				interface: 'select-dropdown',
+				options: {
+					choices: [
+						{ text: '$t:sans_serif', value: 'sans-serif' },
+						{ text: '$t:monospace', value: 'monospace' },
+						{ text: '$t:serif', value: 'serif' },
+					],
+				},
+			},
+			schema: {
+				default_value: 'sans-serif',
+			},
+		},
+		{
+			field: 'previewFont',
+			name: '$t:interfaces.input-rich-text-md.previewFont',
+			type: 'string',
+			meta: {
+				width: 'half',
+				interface: 'select-dropdown',
+				options: {
+					choices: [
+						{ text: '$t:sans_serif', value: 'sans-serif' },
+						{ text: '$t:monospace', value: 'monospace' },
+						{ text: '$t:serif', value: 'serif' },
+					],
+				},
+			},
+			schema: {
+				default_value: 'sans-serif',
+			},
+		},
+		{
+			field: 'toolbar',
+			name: '$t:interfaces.input-rich-text-html.toolbar',
+			type: 'json',
+			schema: {
+				default_value: [
+					'heading',
+					'bold',
+					'italic',
+					'strikethrough',
+					'bullist',
+					'numlist',
+					'blockquote',
+					'code',
+					'link',
+					'table',
+					'image',
+					'empty',
+				],
+			},
+			meta: {
+				width: 'half',
+				interface: 'select-multiple-dropdown',
+				options: {
+					choices: [
+						{
+							value: 'heading',
+							text: '$t:wysiwyg_options.heading',
```

Comment:
Not sure about `t:wysiwyg_options...`
We could use `t:editors...` but we will lose all the translations.
So, for instance, I guess we will keep that way.

Reply:
I think it's fine. We call the "WYSIWYG" interface the _input rich text html_ input, and this markdown one the _input rich text markdown_. Both are "what you see is what you get" style interfaces 👍🏻